### ISSUE TITLE
ask for bootstrap-table-vue.js 's infomation.

### ISSUE THREAD

**Issue by makyo5** [wenzhixin/bootstrap-table#4976]
Hi.
I'm a low level developer. But I have used the bt for 2years.
I use the Vue for develop my web in recent year.
I found there are bootstrap-table-vue.js and bootstrap-table-vue.min.js in the dist folder and I intrested in them. 
I still use the bootstrap-table.js in my vue.js web. 
Can I use the bootstrap-table-vue.js in my vue.js web?? I want to get some docs about how to using the bootstrap-table-vue.js in my vue.js. (I know how to use the bootstrap-table.js in vue web. )

In short, I want for some docs about the bootstrap-table-vue.js. thx.

my email: 2581028207@qq.com

**Comment by UtechtDustin** [comment]
Please check our [documentation page](https://bootstrap-table.com/docs/vuejs/introduction/).
